docs/admin/deployment/onboarding_admins.rst

```diff
@@ -31,60 +31,43 @@ To set up AW2, follow these steps:
    and `set an admin password on the welcome screen <https://tails.boum.org/doc/first_steps/welcome_screen/administration_password/>`__
 2. Ensure that Tails and the SecureDrop version on AW1 are up-to-date.
    If not, update now by following the :ref:`most recent upgrade guide <latest_upgrade_guide>`.
-3. Insert the empty AW2 USB drive.
-4. Launch the Tails installer (**Applications ▸ Tails ▸ Tails Installer**) and install Tails on AW2.
-   This will delete all data on the AW2 USB drive.
-5. Shut down AW1.
-6. Boot AW2.
-7. Configure its persistent volume (**Applications ▸ Tails ▸ Configure persistent volume**).
-   Set a unique passphrase for AW2 and record it securely. Enable all persistence options.
-8. Reboot AW2, unlock its persistent volume, and set an admin password on the welcome screen.
-9. Open the file manager (**Applications ▸ Accessories ▸ Files**).
-10. Insert AW1. It should show up in the list of storage devices in the file manager under
-    a label like "7.0 GB Encrypted". Click the label and enter the drive
-    password when prompted to unlock it.
-11. In a terminal, type the following command:
-
-    ``rsync -a /media/amnesia/TailsData/Persistent/securedrop ~/Persistent``
+3. Log into the *Journalist Interface* using your admin credentials, and create
+    a new user account with admin rights. Record its passphrase securely;
+    you will add it to the password manager on AW2.
 
-    This will copy *only* the ``securedrop`` directory from AW1 to AW2.
+    (You will need to on-board the new admin's 2FA device to complete this step.
+    If this is not possible yet, you can defer it until later.)
+4. Insert the empty AW2 USB drive.
+5. Launch the Tails Cloner (**Applications ▸ Tails ▸ Tails Cloner**).
+   Select the option to **Clone the current Tails.** This will delete all data on the AW2 USB drive.
+6. Check the box marked **Clone the current Persistent Storage.**
+7. Click **Install**.
+8. Choose a unique passphrase for AW2 and record it securely.
+9. Shut down AW1.
+10. Boot AW2 and unlock the Persistent Storage.
+11. Open the KeePassXC database, remove any unneeded credentials from AW1, and store
```

Comment:
This needs clarification - credentials must be deleted, not edited, and the KeepassXC recycle bin cleared, or they'll still be hanging round.